evdns.c

```diff
@@ -5313,6 +5357,98 @@ evdns_result_is_answer(int result)
 	    result != DNS_ERR_SERVERFAILED && result != DNS_ERR_CANCEL);
 }
 
+static void
+evdns_ttl_expired(int result, short what, void *arg)
+{
+	struct evdns_cache *cache = arg;
+	struct evdns_base *base = cache->base; (void)(base);
```

Comment:
Why do you need to suppress `base` usage with `(void)base`?

Reply:
EVDNS_LOCK might be an empty call, so the CI with -Werror will fail with "base is unused variable" otherwise

Reply:
Then we should fix EVDNS_LOCK wrapper to suppress this noise.